docs/src/pages/components/badge.mdx

```diff
@@ -1,2 +1,231 @@
+
+<BadgeWithParentExample />
+
+Here is an example of Badge with a parent component that uses the `isParentHovered` prop:
+
+```jsx hideTitle
+// REACT
+import * as React from 'react';
+// VENDOR
+import styled, { ThemeProvider } from '@xstyled/styled-components';
+// COMPONENTS
+import { Badge, Cart } from '@retailmenot/anchor';
+// THEME
+import { colors, RootTheme } from '@retailmenot/anchor';
+
+const { useState } = React;
+
+const StyledBadgeWithParentExample = styled.div`
+    padding: 2rem 5rem;
+    background-color: ${colors.grapePurchase.base};
+    color: ${colors.white.base};
+    line-height: 1.5rem;
+
+    &:hover {
+        background-color: ${colors.grapePurchase.light};
+    }
+`;
+
+export const BadgeWithParentExample = () => {
+    const [isHovered, setIsHovered] = useState<boolean>(false);
+    const handleOnMouseEnter = () => {
+        setIsHovered(true);
+    };
+    const handleOnMouseLeave = () => {
+        setIsHovered(false);
+    };
+
+    return (
+        <ThemeProvider theme={RootTheme}>
+            <StyledBadgeWithParentExample
+                onMouseEnter={handleOnMouseEnter}
+                onMouseLeave={handleOnMouseLeave}
+            >
+                <Badge
+                    backgroundColor={colors.flashPink.base}
+                    borderColor={colors.grapePurchase.base}
+                    borderColorHover={colors.grapePurchase.light}
+                    count={1}
+                    isParentHovered={isHovered}
+                    offsetBottom={1.1}
+                    offsetLeft={0.6}
+                    overflowCount={9}
+                    showZero={false}
+                    size="small"
+                    textColor={colors.white.base}
+                >
+                    <Cart scale="lg" />
+                </Badge>
+            </StyledBadgeWithParentExample>
+        </ThemeProvider>
+    );
+};
+```
+
+<br />
+
+## API
+
+<table>
+    <thead>
+        <tr>
+            <th>Property</th>
+            <th>Description</th>
+            <th>Type</th>
+            <th>Default</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>backgroundColor</td>
+            <td>The background color of the Badge</td>
+            <td>string | hex code | rgb</td>
+            <td>'error'</td>
+        </tr>
+        <tr>
+            <td>borderColor</td>
+            <td>The color of the border of the Badge</td>
+            <td>string | hex code | rgb</td>
+            <td>'error'</td>
+        </tr>
+        <tr>
+            <td>borderColorHover</td>
+            <td>The color of the border of the Badge on hover</td>
+            <td>string | hex code | rgb</td>
+            <td>'error'</td>
+        </tr>
+        <tr>
+            <td>children</td>
+            <td>Any elements inside the Badge tags. This is what we use to pass Icon components in.</td>
+            <td>Icon</td>
+            <td>-</td>
+        </tr>
+        <tr>
+            <td>className</td>
```

Comment:
Don't worry about documenting `className`. I did it for `Collapse` and I plan on removing it because I will be building out the "All" component section to show things that apply to all components, such as `className`